### PR TITLE
(Feature) Hide wallet and name

### PR DESCRIPTION
**Pull request by clawdius**
Added feature to hide wallet balance and name on the top right corner, user can also toggle only one of them.

Both feature turned off:
image

Both feature turned on:
image

### CODE REVIEW

File: skin.json [hunk 1]
```diff
@@ -1,13 +1,58 @@
...
         {
             "MatchRegexString": "^Steam$",
-            "TargetCss": "Steam SpaceTheme/main.css"
+            "TargetCss": "Steam SpaceTheme/main.css",
+            "Statement": [
+                {
+                    "If": "Wallet Visibility",
+                    "Equals": true,
+                    "True": {
+                        "TargetCss": "Steam SpaceTheme/files/theme/config/WalletVisibility.css"
```
Comment (SpaceEnergy) [inline skin.json:25]: please change all the `config` path to `options`

File: skin.json [hunk 2]
```diff
@@ -1,13 +1,58 @@
...
+                }
+            ]
         },
         {
             "MatchRegexString": "Friends List",
             "TargetJs": "friends.custom.js",
             "TargetCss": "friends.custom.css"
+        },
+        {
+            "MatchRegexString": "^Account",
```
Comment (SpaceEnergy) [inline skin.json:45]: and here i don't know which side this is applied to (maybe just tell me where it will be patched)

**Review by SpaceEnergy — CHANGES_REQUESTED** [review]
please change these things

**Comment by clawdius** [comment]
1. Changed `config` into `options`
2. The `^Account` regex refer to this window when user clicked at profile menu
image

**Comment by SpaceEnergy** [comment]
ah ok, all right

**Review by SpaceEnergy — APPROVED** [review]

**Pull request merged**